cluster/gce/configure.sh

```diff
@@ -158,29 +158,30 @@ fi
 log_level="${CONTAINERD_LOG_LEVEL:-"info"}"
 max_container_log_line="${CONTAINERD_MAX_CONTAINER_LOG_LINE:-16384}"
 cat > ${config_path} <<EOF
+version = 2
 # Kubernetes requires the cri plugin.
-required_plugins = ["cri"]
+required_plugins = ["io.containerd.grpc.v1.cri"]
```

Comment:
ouch...  

is this prefix necessary now?

Reply:
Yes, it is now.

Reply:
I see it now thx. Ouch but thx.